Framework/src/CcdbDatabase.cxx

```diff
@@ -181,6 +181,7 @@ void CcdbDatabase::storeMO(std::shared_ptr<const o2::quality_control::core::Moni
   metadata["qc_detector_name"] = mo->getDetectorName();
   metadata["qc_task_name"] = mo->getTaskName();
   metadata["ObjectType"] = mo->getObject()->IsA()->GetName(); // ObjectType says TObject and not MonitorObject due to a quirk in the API. Once fixed, remove this.
+  metadata["Run"] = std::to_string(mo->getRunNumber());
```

Comment:
Why "Run" instead of "RunNumber"? If we add more fields like "RunType", "RunDuration", then it will be inconsistent.

Reply:
That is a very good point. I had asked Costin about it but I think that we overlooked this problem. I will indeed move to "RunNumber".